nle/nethack/nethack.py

```diff
@@ -224,3 +224,6 @@ def get_current_seeds(self):
 
     def in_normal_game(self):
         return self._pynethack.in_normal_game()
+
+    def how_done(self):
```

Comment:
will this crash if its done outside of the end game?

Reply:
No, but it will return `game_end_types.DIED` which corresponds to `0`.